great_expectations_cloud/agent/models.py

```diff
@@ -10,7 +10,8 @@
 
 class AgentBaseModel(BaseModel):  # type: ignore[misc] # BaseSettings is has Any type
     class Config:
-        extra: str = Extra.ignore
+        # 2024-03-04: ZEL-501 Strictly enforce models for handling outdated APIs
+        extra: str = Extra.forbid
```

Comment:
Have you talked with @JennyTee about this change?
I'm all for it but want to make sure she's in the loop.

Reply:
:point_up:

Reply:
Thanks, @Kilo59, much appreciated! Glad that we are changing it back now that we can more gracefully handle mismatched event models. 🎉